sdk/src/Services/S3/Custom/Internal/AmazonS3RetryPolicy.cs

```diff
@@ -116,6 +120,17 @@ public partial class AmazonS3RetryPolicy : DefaultRetryPolicy
                     }
                 }
             }
+            else
```

Comment:
This is a behavior I think we need a team discussion on.  Before, we didn't retry on any of these webexceptions for S3, but now we are.  Could you get @gokarnm's opinion on this?